CHANGELOG.md

```diff
@@ -2,6 +2,116 @@
+
+- Code Security: add the boto package to the IAST patching denylist.
+- celery: changes `error.message` span tag to no longer include the traceback that is already included in the `error.stack` span tag.
+- CI Visibility: fixes source file information that would be incorrect in certain decorated / wrapped scenarios and forces paths to be relative to the repository root, if present.
+- LLM Observability: This resolves a typing hint error in the `ddtrace.llmobs.utils.Documents` helper class constructor where type hints did not accept input dictionaries with integer or float values.
+- LLM Observability: This fix resolves an issue where the OpenAI, Anthropic, and AWS Bedrock integrations were always setting `temperature` and `max_tokens` parameters to LLM invocations. The OpenAI integration in particular was setting the wrong `temperature` default values. These parameters are now only set if provided in the request.
+- redis: This fix resolves an issue in the redis exception handling where an UnboundLocalError was raised instead of the expected BaseException.
+- ASM: This fix resolves an issue where the <span class="title-ref">requests</span> integration would not propagate when apm is opted out (i.e. in ASM Standalone).
+- profiling: Fixes an issue where task information coming from echion was encoded improperly, which could segfault the application.
+- tracing: fixes a potential crash where using partial flushes and `tracer.configure()` could result in an IndexError
+- flask: Fix scenarios when using flask-like frameworks would cause a crash because of patching issues on startup.
+- wsgi: Ensures the status of wsgi Spans are not set to error when a `StopIteration` exception is raised marked the span as an error. With this change, `StopIteration` exceptions in this context will be ignored.
+- langchain: tag non-dict inputs to LCEL chains appropriately. Non-dict inputs are stringified, and dict inputs are tagged by key-value pairs.
+- langchain: This fixes an issue of langchain patching errors due to the `langchain-community` module becoming an optional dependency in `langchain>=0.2.0`. The langchain integration now conditionally patches `langchain-community` methods if it is available. See the langchain integration :ref: <span class="title-ref">docs\<langchain\></span> for more details.
+
+- ASM: This fix resolves an issue where an org could not customize actions through remote config.
+- ASM: protect against potentially returning `None` when tainting a gRPC message.
+- botocore: This fix adds additional key name checking and appropriate defaults for responses from Cohere and Amazon models.
+- This fix resolves an issue where importing `asyncio` after a trace has already been started will reset the currently active span.
+- CI Visibility: fixes that traces were not properly being sent in agentless mode, and were otherwise not properly attached to the test that started them
```

Comment:
```suggestion
- CI Visibility: Fixes traces that were not properly being sent in agentless mode, and were otherwise not properly attached to the test that started them
```